src/node.cc

```diff
@@ -195,10 +195,27 @@ void SignalExit(int signo, siginfo_t* info, void* ucontext) {
 }
 #endif  // __POSIX__
 
+#if HAVE_INSPECTOR
+class InspectorPauseResetter {
+ public:
+  explicit InspectorPauseResetter(inspector::Agent* agent) : agent_(agent) {}
+  ~InspectorPauseResetter() {
+     if (agent_->options().break_node_first_line) {
```

Comment:
This is now ambiguous because it breaks on the first line of each script, not just the first line of the first script (modulo per-context scripts)

Reply:
@joyeecheung I think that is @danbev’s intention here, if I read the PR description correctly?

Reply:
@joyeecheung Yeah, that is the intention here. Sorry if that was not clear from the description. I've added a https://github.com/nodejs/node/pull/29203#issuecomment-524715400 which hopefully sheds some light on this. Thanks